gpt_researcher/skills/image_generator.py

```
@@ -0,0 +1,767 @@
+            report: The original report markdown.
+            images: List of generated image info.
+            suggestions: Original suggestions with section info.
+            
+        Returns:
+            Modified report with embedded images.
+        """
+        lines = report.split('\n')
+        
+        # Create a mapping of section headers to images
+        section_to_image = {}
+        for img, sug in zip(images, suggestions):
```

Ruff (0.14.14)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)